examples/mnist_template.py

```diff
@@ -20,6 +21,7 @@
 @dataclass
 class CommonArgparseArgs:
```

Comment:
I hate that class name :)

Reply:
yeah don't worry it's going away. But that brings up a different question:

There will be a lot of miscellaneous args like the ones in `ExportedArgparseArgs` that I'm thinking it would be useful to have a class which sets sane defaults for. (usually people create a ton of argparse logic in __main__ for these). I was thinking of one paradigm where one could create a class from the contents of the argparse code for any specfic project, hence "`Exported`".

One option is to handwrite a generic one, since I think each of these values are too strewn about across pytorch.

Reply:
Maybe @tkornuta-nvidia has some throughts on this one too (pertaining to how they are using hydra in neom)?

Reply:
I gut feeling is that for common stuff, especially for a simple example - we can just drop them in the top level of the primary config directly.

For cases like Nemo, that's an excellent question and I actually already made a suggestion along those lines to @tkornuta-nvidia [here](https://github.com/facebookresearch/hydra/issues/1049).

Reply:
yeah, somehow I do not see that as part of Hydra-Torch configs...

But generally I agree with the need of having some kind of "AppConfig" containing parameters from argparse.

Actually I find it linked with [Feature Request 1048](https://github.com/facebookresearch/hydra/issues/1048)

Reply:
> I gut feeling is that for common stuff, especially for a simple example - we can just drop them in the top level of the primary config directly.
For the Basic tutorial, this is the solution. In the Intermediate one, we can decide if there should be an additional `ArgsConf` or they should stay flat on the top level.